eclipse-collections-code-generator/src/main/resources/test/set/immutable/immutablePrimitivePrimitiveMapKeySetTest.stg

```diff
@@ -5,7 +5,7 @@ isTest() ::= "true"
 
 skipBoolean() ::= "true"
 
-targetPath() ::= "org/eclipse/collections/impl/map/mutable/primitive"
+targetPath() ::= "org/eclipse/collections/impl/map/immutable/primitive"
```

Comment:
Should probably get these 3 back into the mutable package - incorrect from semantics perspective but this is where their main classes are so would be consistent